sparts/tasks/dbus.py

```diff
@@ -158,8 +160,13 @@ def initTask(self):
         assert self.bus_name is not None, \
             "You must pass a --{task}-bus-name"
 
+    def getBus(self):
```

Comment:
minor nit,  it's not clear from `getBus` that this is instantiating a new private Bus (vs. returning the existing local).  Also prefer an underscore prefix for things that are not meant to be called externally.

Call this `_makeBus` or `_getNewBus` or `_createBus` instead (I like `_makeBus` the most)